atomate/vasp/workflows/base/neb.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+This module defines the Climbing Image Nudged Elastic Band (CI-NEB) workflow.
+"""
+
+import os
+from datetime import datetime
+
+from pymatgen.core import Structure
+from pymatgen_diffusion.neb.io import get_endpoints_from_index
+from fireworks.core.firework import Workflow
+
+from atomate.vasp.fireworks.core import NEBFW, NEBRelaxationFW
+from atomate.utils.utils import get_logger
+
+__author__ = "Hanmei Tang, Iek-Heng Chu"
+__email__ = 'hat003@eng.ucsd.edu, ihchu@eng.ucsd.edu'
+
+logger = get_logger(__name__)
+spec_orig = {"wf_name": "CINEB",
```

Comment:
I am not sure what this is? It seems to have a lot of code that is (i) tailored for your cluster and (ii) anyway, has nothing to do with the materials science problemlike ppn or _queueadapter.